src/api/app/controllers/source_attribute_controller.rb

```diff
@@ -68,7 +68,7 @@ def update
 
       attrib.container = @attribute_container
 
-      raise APIError, message: attrib.errors.full_messages.join('\n'), status: 400 unless attrib.valid?
+      raise APIError, attrib.errors.full_messages.join('\n') unless attrib.valid?
```

Comment:
This is because we added the initializer in the APIError. This way of raising an error was working but it's not the standard way of raising an exception according to the documentation.

Reply:
This change isn't needed anymore.

Reply:
Done.